include/gz/gui/config.hh.in

```diff
@@ -8,7 +8,7 @@
 #define IGNITION_GUI_VERSION "${PROJECT_VERSION}"
 #define IGNITION_GUI_VERSION_FULL "${PROJECT_VERSION_FULL}"
 
-#define IGNITION_GUI_VERSION_HEADER "Ignition ${IGN_DESIGNATION}, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2017 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
+#define IGNITION_GUI_VERSION_HEADER "Gazebo ${IGN_DESIGNATION}, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2017 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
```

Comment:
Should we update the date to `2022` since we are going from Ignition --> Gazebo?
```suggestion
#define IGNITION_GUI_VERSION_HEADER "Gazebo ${IGN_DESIGNATION}, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2022 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
```

Reply:
humm I hadn't thought about that. I believe the year should refer to the copyright by OSRF, not necessarily tied to the name of the library, so I say we keep it with the original date.